legal-api/src/legal_api/models/__init__.py

```diff
@@ -88,6 +93,7 @@
     'Review',
     'ReviewResult',
     'ReviewStatus',
+    'Role',
```

Comment:
How about calling this AuthorizedRole?

Reply:
PS - Today we will find authorized actions by role. However, this may evolve in the future (eg, find by account id).

I don't want to make it look like this is a the complete list of roles. It's only the roles we care about for authorized actions.